docs/rules/jsx-wrap-multilines.md

```diff
@@ -6,14 +6,17 @@ Wrapping multiline JSX in parentheses can improve readability and/or convenience
 
 ## Rule Details
 
-This rule optionally takes a second parameter in the form of an object, containing places to apply the rule. By default, all the syntax listed below will be checked, but these can be explicitly disabled. Any syntax type missing in the object will follow the default behavior (become enabled).
+This rule optionally takes a second parameter in the form of an object, containing places to apply the rule. By default, all the syntax listed below will be checked except the logical expressions and JSX attributes, but these can be explicitly disabled. Any syntax type missing in the object will follow the default behavior (become enabled).
 
 There are the possible syntax available:
 
 * `declaration`
 * `assignment`
 * `return`
 * `arrow`
+* `condition`
+* `logical` (not enabled by default)
+* `attr` (not enabled by default)
```

Comment:
this should probably be "prop" - in jsx, elements have props, and "attributes" are something HTML elements have (which jsx isn't)

Reply:
Agreed!
I've renamed to `prop`